src/openrct2-ui/UiContext.Linux.cpp

```diff
@@ -295,14 +295,14 @@ namespace OpenRCT2::Ui
                 case DIALOG_TYPE::ZENITY:
                 {
                     auto sb = StringBuilder();
-                    sb.Append(reinterpret_cast<utf8*>(
-                        String::Format("zenity --list --column '' --width=%d --height=%d", width, height)));
+                    sb.Append(const_cast<const char*>(
```

Comment:
Add a `std::string_view` overload to `StringBuilder`, casting should not be necessary.